synapse/config/logger.py

```diff
@@ -81,11 +81,18 @@
 """
 )
 
+NO_LOG_FILE_ERROR = """\
```

Comment:
I find this name confusing, I would understand it as an error you raise when there's no `log_file`/`--log-file` provided (but it's actually the opposite). Maybe rename it in `LOG_FILE_ERROR`?

Reply:
yup, fair. fixed.